docs/source/troubleshooting.rst

```diff
@@ -45,3 +45,9 @@ Removing the volume for RHEL 6 or Centos 6
 Removing the volume for RHEL 7, Centos 7, Fedora 27, or Fedora 28
 
 # docker volume rm qpc-data
+
+Adding Active Directory Credentials
+-----------------------------------
+Active directory accounts may structure their usernames to include the domain separated by a backslash e.g. ``Domain\Username``. However, this structure will result in issues due to the backslash being used as an escape character in both Shell and Python. In order to add a username with a backslash present, the backslash will need to be escaped with another backslash ``\\``::
```

Comment:
Again this could apply to any value (not just username).  We could username and Active directory as a concrete example.

Modify this so it addresses the general problem then ends with a concrete example containing ActiveDirectory and username.